crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -1259,23 +1148,60 @@ impl actor::HcP2p for HolochainP2pActor {
 
     fn send_validation_receipts(
```

Comment:
I thought we were going to change how this works and there's an issue about that? Should this one be a stub for now?

Reply:
We're going to change how we get the notification that we *need* to send a validation receipt. But we still need this same call to actually send them once we know we need to : )

Reply:
Fair enough, I guess we don't want to do that as a direct response because it'll require us to run validation so it probably needs to be queued work